.github/workflows/docker.yaml

```diff
@@ -29,6 +29,46 @@ jobs:
   build-and-push:
     runs-on: ubuntu-latest
     steps:
+      - id: release-params
+        name: Look up the Github release by tag/name and prepare a list of tags to be attached to the Docker image
+        run: |
+          python3 <<EOF
+          import json
+          import urllib.error as e
+          import urllib.request as r
+
+          # look up the latest stable release (if any)
+          try:
+            latest_release = json.load(r.urlopen("https://api.github.com/repos/xsnippet/xsnippet-api/releases/latest"))
```

Comment:
It's a bit inconvenient that the tag for xsnippet-web is in ansible playbooks, and for the api is in GitHub actions. Hopefully, it will gone soon.

Reply:
Right, but we would still need to make the tag configurable if we want to be able to deploy something besides the latest release?